### ISSUE TITLE
Image URL keeps the way it is

### ISSUE THREAD

**Issue by emisjerry** [mvdkwast/obsidian-copy-as-html#15]
An images URL don't convert to SVG, keeps the URL.

For example, my Markdown is:

```
![GitHub](https://fastly.jsdelivr.net/gh/emisjerry/upgit@master/20221670655020000nmd1lp.png)
```

Please keep the origianl Markdown, do not convert to SVG.

**Comment by mvdkwast** [comment]
Thanks for reporting, this looks like a bug indeed. This should be an easy fix.

**Comment by mvdkwast** [comment]
Ok, the real issue here is that external pictures are always embedded, which may or may not be what the user wants. Nothing is "converted to SVG", but images are downloaded and transformed to inline data.

I think the right approach here is to add an option to choose whether external links are embedded or not.

**Comment by mvdkwast** [comment]
Fixed by 5d58dcbc6f9ba709e4daecaa73d985d1e9f428fa
